public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -285,6 +285,7 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel)
     body: new SceneGridLayout({
       isLazy: true,
       children: createSceneObjectsForPanels(oldModel.panels),
+      $behaviors: [trackIfEmpty],
```

Comment:
@kaydelaney  - I'm also moving the behavior to the grid as i think it belongs here, also makes the behavior simpler.